apps/extension/src/ui/apps/dashboard/routes/Settings/GeneralPage.tsx

```diff
@@ -26,11 +36,23 @@ export const GeneralPage = () => {
   const [identiconType, setIdenticonType] = useSetting("identiconType")
   const [allowNotifications, setAllowNotifications] = useSetting("allowNotifications")
   const [spiritClanFeatures, setSpiritClanFeatures] = useSetting("spiritClanFeatures")
+  const [hasRuntimeReloadFn, runtimeReload] = useRuntimeReload(ANALYTICS_PAGE)
 
   return (
     <DashboardLayout centered>
       <HeaderBlock title={t("General")} text={t("General settings")} />
       <div className="mt-16 flex flex-col gap-4">
+        {hasRuntimeReloadFn ? (
+          <Setting
+            iconLeft={RefreshCwIcon}
+            title={t("Reload Talisman")}
+            subtitle={t("Close all wallet tabs and restart the background script")}
```

Comment:
Users might not know what a background script is, or when they should do this, so how about:
```suggestion
            subtitle={t("Closes and restarts Talisman, can help to resolve error states.")}
```

Reply:
Awesome! I made a slight change so that the grammar is consistent with the other descriptions on this page:

![Screenshot 2023-11-30 at 15 10 35](https://github.com/TalismanSociety/talisman/assets/2741569/d446658b-80de-4652-96ce-816d46e725f6)